src/drawings/falling-ball/FallingBall.jsx

```diff
@@ -3,16 +3,18 @@
 import { CanvasBuilder } from '@/components/Canvas'
 import { Physics, Color } from '@/constants'
 
+const HEIGHT = 200
```

Comment:
It doesn't need to be 200px. This change was just to make sure it actually worked. I can revert to 100px if we care